README.md

```diff
@@ -0,0 +1,85 @@
+kenya-china-square Planning,
+
+`author`
+Peter-Kibet -
+
+themathematcs - chriznganga69@gmail.com
+
+program defination:
+"Our app is a cross-border e-commerce platform that allows buyers and sellers from two different countries to connect with each other and buy and sell a variety of goods,
+including electronics, clothing, and automobiles.
+With our app, users can browse and search for items, view seller profiles, and communicate directly with sellers to negotiate prices and arrange for shipping and delivery.
+Our app provides a secure and convenient platform for cross-border transactions,
+making it easy for buyers and sellers to conduct business with each other regardless of their location.
+
+statistics of the advancement in technology possing a great opotunity in an online market:
```

Comment:
Please use the Microverse README template and always make sure to include a table of contents in your README. Generally, your README file is very informative.

Reply:
- [ ] Please can you explain further what you meant by **Generally, your README file is very informative?** Perhaps, you want to write **Generally, your README file supposed to be informative** I guess. Kindly rephrase the statement (i.e. the essence of using Microverse README template and having table of contents in the README). Thanks. 🙏